src/util-exception-policy.c

```diff
@@ -88,7 +88,11 @@ enum ExceptionPolicy ExceptionPolicyParse(const char *option, const bool support
             policy = EXCEPTION_POLICY_IGNORE;
             SCLogConfig("%s: %s", option, value_str);
         } else {
-            SCLogConfig("%s: ignore", option);
+            FatalErrorOnInit(SC_ERR_INVALID_ARGUMENT,
+                    "\"%s\" is not a valid exception policy value. Valid options are drop-flow, "
+                    "pass-flow, bypass, drop-packet or pass-packet. For more infomation, please "
+                    "check the Suricata documentation.",
```

Comment:
thinking about this again, I wonder if we should reference the docs. Its not something we really do (much) elsewhere. Why would we do so here? Or, should we do so everywhere?

Reply:
I think that... if the information given can be considered enough for most users to understand what they have to next, then, indeed referring to the documentation may be too much.

On the other hand, it is a way of telling them that we do have more detailed information on topics that may need elaboration. What is a good balance? 🤔

Reply:
Also wondering if we should then mention a specific doc/url, instead of generally saying "go read the docs".

In reality most ppl will just google it most likely.

Reply:
I'll remove that bit :P